docs/release-notes.rst

```diff
@@ -6,6 +6,99 @@
+   create`` and ``det user change-password`` CLI commands.
+
+   When creating non-remote users at the CLI with ``det user create``, setting a password is now
+   mandatory. You can set the password interactively by following the prompts during user creation
+   or non-interactively with the ``--password`` option.
+
+**New Features**
+
+-  Kubernetes: In the enterprise edition, add the ability to set up the Determined master service on
+   one Kubernetes cluster and manage workloads across different Kubernetes clusters. Additional
+   non-default resource managers and resource pools are configured under the
+   ``additional_resource_managers`` section (additional resource managers are required to have at
+   least one resource pool defined). Additional resource managers and their resource pools must have
+   unique names. For more information, visit :ref:`master configuration <master-config-reference>`.
+   Support for notebooks and other workloads that require proxying is under development.
+
+-  API/CLI/WebUI: In the enterprise edition, route any requests to resource pools not defined in the
+   master configuration to the default resource manager, not any additional resource manager, if
+   defined.
+
+-  Configuration: In the enterprise edition, add an ``additional_resource_managers`` section that
+   can define multiple resource managers following the same patteroas ``resource_manager``. Add
+   ``name`` and ``metadata`` fields to individual resource manager definitions.
+
+-  WebUI: In the enterprise edition, add the ability to view resource manager name for resource
+   pools.
+
+**Improvements**
+
+-  Configuration: The master configuration parameter ``observability.enable_prometheus`` now
+   defaults to ``true``. Consequently, Prometheus endpoints are enabled by default, which does not
+   affect clusters that do not use Prometheus.
+
+**Bug Fixes**
+
+-  Since version 0.27.1, a bug has affected the selection functionality in the Experiments page.
+   This bug may cause actions to be applied to more experiments than are visibly selected. For
+   example, when using the **Select All > Actions > Move** sequence to transfer all experiments from
+   one project to another, the action may inadvertently include experiments not only from the
+   targeted project but also from other projects you have permissions to edit. We urge all users on
+   the affected versions to upgrade as soon as possible.
+
+   -  There is a risk of data loss if, when attempting to delete a set of experiments, the action
+      inadvertently deletes a larger set than intended.
+
+   -  For RBAC (Role-Based Access Control) (an Enterprise-edition feature) users, there is a risk of
+      a permissions leak if moving experiments from one project to another inadvertently includes
+      experiments from other workspaces.
+
+   -  This issue affects all bulk actions including delete, move, archive, unarchive, resume, pause,
+      kill, stop, and viewing in TensorBoard.
```

Comment:
```suggestion
      kill, stop, and view in TensorBoard.
```

Reply:
(Or alternately go the other way and make the whole list "deleting, saving, moving, ...".)